tests/integrate_test/cita_jsonrpcTest.sh

```diff
@@ -34,6 +34,11 @@ invalid_data=-32600
 invalid_jsonrpc_method=-32601
 
 ./cita_start.sh
+
+# Check JSON-RPC CORS: Access-Control-Allow-Origin should be existed.
+has_cors=$(curl -i -X POST -d '{"jsonrpc":"2.0","method":"peerCount","params":[],"id":2}'  $IP:$PORT 2>/dev/null | grep -ic "^access-control-allow-origin: ")
```

Comment:
Suddenly, I realized, the test request should be:
```
curl -i -X OPTIONS $IP:$PORT 2>/dev/null | grep -ic "^access-control-allow-origin: "
```

All are OK, but this is better.